### PR TITLE
Minor OpenAPI improvements: documentation, status codes

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: components/openapi/src/main/scala/pl/touk/nussknacker/openapi/OpenAPIsConfig.scala [hunk 1]
```diff
@@ -3,12 +3,14 @@ package pl.touk.nussknacker.openapi
...
 import io.swagger.v3.oas.models.PathItem.HttpMethod
 import net.ceedubs.ficus.readers.{ArbitraryTypeReader, ValueReader}
+import sttp.model.StatusCode
 
 import java.net.URL
 import scala.util.matching.Regex
 
 case class OpenAPIServicesConfig(//by default we allow only GET, as enrichers should be idempotent and not change data
                                  allowedMethods: List[String] = List(HttpMethod.GET.name()),
+                                 codesToInterpretAsEmpty: List[Int] = List(StatusCode.NotFound.code),
```
[the remark this answers is not in the document]
Comment (mproch) [reply components/openapi/src/main/scala/pl/touk/nussknacker/openapi/OpenAPIsConfig.scala:13]: It's parsed as HOCON, so you can configure it - just like allowedMethods

File: components/openapi/src/main/scala/pl/touk/nussknacker/openapi/package.scala [hunk 2]
```diff
@@ -42,14 +42,23 @@ package openapi {
...
                                              categories: List[String],
                                              documentation: Option[String],
                                              pathParts: List[PathPart],
                                              parameters: List[SwaggerParameter],
                                              responseSwaggerType: Option[SwaggerTyped],
                                              method: String,
                                              servers: List[String],
-                                             securities: List[SwaggerSecurity])
+                                             securities: List[SwaggerSecurity],
+                                             codesToInterpretAsEmpty: List[Int]
```
[the remark this answers is not in the document]
Comment (mproch) [reply components/openapi/src/main/scala/pl/touk/nussknacker/openapi/package.scala:54]: https://github.com/TouK/nussknacker/pull/3719/files#diff-fff32da0a1097482b496b16186f8e47b96fedfc27e98dc5169c093be74585829R15 => it's used here, so we have to pass it. Anyway, I think some test should be provided... I'll add one

[the remark this answers is not in the document]
Comment (arkadius) [reply components/openapi/src/main/scala/pl/touk/nussknacker/openapi/package.scala:54]: Ok, I understand but IMO SwaggerService is not a good class to pass it. At first glance this class looks like our representation of swagger service. codesToInterpretAsEmpty isn't available in swagger definition, it is our "configuration", like rootUrl. The current approach will make use of this class for some mocking, transferring data etc harder.

[the remark this answers is not in the document]
Comment (mproch) [reply components/openapi/src/main/scala/pl/touk/nussknacker/openapi/package.scala]: Oh well, you're right. I moved it closer to creation of enricher itself, ended up moving a few more things...

**Review by arkadius — APPROVED** [review]
Good job

**Pull request merged**